lib/editor/actions/map/locationStrategies.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import type { dispatchFn, getStateFn } from '../../../types/reducers'
+import { updatingLocationShape } from '../active'
+import type { GeometryType, LocationShape } from '../../../types'
+
+/**
+ * Updates the location shape (Polygon, polyline, multi polygon or polyline)
+ */
+export function updateLocationShape (
```

Comment:
@philip-cline any thoughts

Reply:
Binh's right I think this could go (I think it was left over from when this function was more complicated) and be replaced with a call to `dispatch(updatingLocationShape({geometryType: 'geometry-type', shapes: [ ...] }))`